### PR TITLE
Community update

### LINKED ISSUE #68: community page obsolete
I think https://pandas.pydata.org/community.html needs an update.

It should probably not mention IRC (right?) while it should mention Gitter and pydata@googlegroups.com .

### PR DESCRIPTION
**Pull request by TomAugspurger**
Closes #68

### CODE REVIEW

**Comment by TomAugspurger** [comment]
cc @pandas-dev/pandas-core 

Anything we'd like to change on how we handle community conversations?

- I wasn't sure where (if anywhere) to mention the pydata google group.
- I need to add a section for the ~monthly developer meetings.

File: community.rst [hunk 1]
```diff
@@ -2,16 +2,30 @@
...
+   for *usage* questions (like "How do I do *X* in pandas?")
+2. Use the `GitHub issue tracker <https://github.com/pandas-dev/pandas>`__ for
+
+   - **Bug reports** (like "``DataFrame.head()`` returns 6 rows, when it should return 5.")
+   - **Documentation issues** (like "I found this section unclear")
+   - **Feature requests** (like "I think the DataFrame repr should include a pandas emoji next to the shape.")
+
+3. Use the `pandas dev mailing list <https://mail.python.org/mailman/listinfo/pandas-dev>`__ for
+   longer-form discussion items
+   - How should we have discussions about pandas?
```
Comment (jorisvandenbossche) [inline community.rst]: might be there needs to be a blank line here

File: community.rst [hunk 2]
```diff
@@ -2,16 +2,30 @@
...
+1. Use the `#pandas tag on StackOverflow <http://stackoverflow.com/questions/tagged/pandas>`__
+   for *usage* questions (like "How do I do *X* in pandas?")
+2. Use the `GitHub issue tracker <https://github.com/pandas-dev/pandas>`__ for
+
+   - **Bug reports** (like "``DataFrame.head()`` returns 6 rows, when it should return 5.")
+   - **Documentation issues** (like "I found this section unclear")
+   - **Feature requests** (like "I think the DataFrame repr should include a pandas emoji next to the shape.")
+
+3. Use the `pandas dev mailing list <https://mail.python.org/mailman/listinfo/pandas-dev>`__ for
+   longer-form discussion items
```
Comment (jorisvandenbossche) [inline community.rst]: I would be a bit more broad here in the description what is welcome to discuss here, as I find the list below a bit limiting. Basically anything related to the pandas development? (both community, development workflow as implementation, API design, roadmap, ...) 
Or would that steer people too much towards mailing pandas-dev when a github issue is more appropriate? (although I don't have the feeling that is currently a problem)

File: community.rst [hunk 3]
```diff
@@ -2,16 +2,30 @@
...
+   - What items are on pandas roadmap?
+   - I'm hosting a pandas sprint.
+
+   This is also used to announce pandas releases and developer meetings.
+
+4. Use the `pandas gitter <https://gitter.im/pydata/pandas>`__ for quick feedback on
+   *development* issues
+
+   - I'm having trouble setting up the development environment.
+   - I messed up my git branches, can someone take a look?
```
Comment (jorisvandenbossche) [inline community.rst]: Maybe explicitly add a `- ...` to be clear this is not a limiting list? (and above as well)

**Comment by jorisvandenbossche** [comment]
> Anything we'd like to change on how we handle community conversations? 
> This thread on the PyData mailing list: https://groups.google.com/forum/#!topic/pydata/3IqSwbifVhM raises some proposals. In particular, theres a desire from at least two participants on that mailing list for something similar to Julia's discourse.

@TomAugspurger I would keep actual discussion on changes to our communication channels for on the mailing list (where it started), don't think pandas-website is the best place for that. And let's keep it here about better describing the *current* communication channels?

**Comment by jorisvandenbossche** [comment]
> @wesm I notice that "Pandas-dev" doesn't have a description in the index at https://mail.python.org. Are you an admin on that? Could we update that to reflect what we view that list for? I think that the "There isn't really any place for users to interact with developers to discuss longer term plans (e.g. version 0.25, version 2.0)" from the original post to the PyData mailing list would fit well on the dev mailing list.

I can also update that it seems (tested by changing it to "Pandas development mailing list"). If there are other concrete suggestions, let me know.

**Comment by toobaz** [comment]
Don't we want to mention ``pydata@googlegroups.com`` ? It is mostly used for pandas, and while certainly it is not very active, and most messages could maybe be legitimate questions on SO... it still looks to me it's doing its job.

**Comment by TomAugspurger** [comment]
> I would keep actual discussion on changes to our communication channels for on the mailing list (where it started), don't think pandas-website is the best place for that. 

Agreed. I meant, "what do we take from that discussion".

Anyone want to attempt a description of the pydata@googlegroups.com mailing list, and how it differs from pandas-dev?

**Comment by jorisvandenbossche** [comment]
Shall we merge this? As it is certainly an improvement (or wait until after the meeting?)

**Comment by TomAugspurger** [comment]
Probably worth merging for now.

**Pull request merged**